README.md

```diff
@@ -84,17 +84,34 @@ After we have installed the virt plugin we can use it:
 
 ## Creating krew packages after a new KubeVirt release 
 
-Example: prepare a kubectl krew release for a new release
+**Note: a [prow job](https://github.com/kubevirt/project-infra/blob/cd56918c29690a1cda96d91a781b948f105c1dd6/github/ci/prow-deploy/files/jobs/kubevirt/kubectl-virt-plugin/kubectl-virt-plugin-periodics.yaml#L2) periodically executes the below script**
```

Comment:
Makes sense to link to main instead of a specific blob?

Reply:
I rather use the `Permalink` in general, since I am pointing to a specific line. This is to avoid pointing to the wrong thing if using `tree/main` and then the file is changed.

Reply:
ACK